app/helpers/solidus_friendly_promotions/admin/benefits_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module SolidusFriendlyPromotions
+  module Admin
+    module BenefitsHelper
+      def options_for_benefit_calculator_types(benefit)
+        calculators = benefit.available_calculators
+        options = calculators.map { |calculator| [calculator.model_name.human, calculator.name] }
+        options_for_select(options, benefit.calculator_type.to_s)
+      end
+
+      def options_for_benefit_types(benefit)
+        benefits = SolidusFriendlyPromotions.config.actions
+        options = benefits.map { |action| [action.model_name.human, action.name] }
+        options_for_select(options, benefit&.type&.to_s)
+      end
+    end
```

Comment:
Don't you also need `benefits_by_level`?

Reply:
No, the need for that has been refactored away in a previous PR :)